Frontend/lib/models/club.dart

```diff
@@ -17,12 +17,15 @@ class Club {
     required this.bannerImageURL,
   });
 
-  Club.fromJson(Map<String, dynamic> dataMap)
-      : id = dataMap['id'],
-        name = dataMap['name'],
-        address = dataMap['address'],
-        postalCode = dataMap['postal_code'],
-        city = dataMap['city'],
-        ownerID = dataMap['owner_id'],
-        bannerImageURL = dataMap['banner_image_url'];
+  factory Club.fromJson(Map<String, dynamic> dataMap) {
+    return Club(
+      id: dataMap['id'],
+      name: dataMap['name']?.toString() ?? '',
```

Comment:
Is het nodig om overal toString achter te zetten, het zijn allemaal toch al strings lijkt mij?